ArduPlane/quadplane.cpp

```diff
@@ -1699,14 +1699,15 @@ void SLT_Transition::update()
         // after airspeed is reached we degrade throttle over the
         // transition time, but continue to stabilize
         const uint32_t transition_timer_ms = now - transition_low_airspeed_ms;
-        if (transition_timer_ms > (unsigned)quadplane.transition_time_ms) {
+        float trans_time_ms = constrain_float(quadplane.transition_time_ms,2000,20000);
```

Comment:
I don't think we should constrain on the upper side. You might as well take the opportunity to const it too.
```suggestion
        const float trans_time_ms = MAX(quadplane.transition_time_ms, 2000);
```

Reply:
will add const....upper limit was Tridges request

Reply:
Do we have a example of a tool long time causing a issue?

A arming check would make me happier, that makes it very clear that we consider the old value invalid and there will be a change in behavior. I much prefer the "why won't my vehicle arm" issues to the "may vehicle just did something different to what it did before and then crashed" issues.

Reply:
too long is a battery consumption issue at least

Reply:
currently its 30sec in the param, maybe just match that?